docs/TVM_EP.md

```diff
@@ -138,18 +228,18 @@ so.graph_optimization_level = onnxruntime.GraphOptimizationLevel.ORT_DISABLE_ALL
 tvm_session = onnxruntime.InferenceSession(model_path, sess_options=so, providers=["TvmExecutionProvider"], provider_options=po)
 ```
 
-### Using precompiled model
+### **Using precompiled model**
```

Comment:
Why adding **?

Reply:
Subtitle blends in with the text around and I add bold style to highlight it